flink-core/src/main/java/org/apache/flink/api/connector/source/util/ratelimit/RateLimiterStrategy.java

```diff
@@ -27,14 +27,14 @@
  * A factory for {@link RateLimiter RateLimiters} which apply rate-limiting to a source sub-task.
  */
 @Experimental
-public interface RateLimiterStrategy extends Serializable {
+public interface RateLimiterStrategy<S> extends Serializable {
 
     /**
      * Creates a {@link RateLimiter} that lets records through with rate proportional to the
```

Comment:
nit: don't we mean inversely proportional to the parallelism? Though this is a bit mathematical. Maybe 

` Creates a {@link RateLimiter} that limits the rate of records going through. When there is parallelism, the limiting rate is evenly reduced per subtask, such that all the sub-tasks limiting rates equals the cumulative limitting rate.  `

Reply:
Addressed.